lib/livebook_web/cookies.ex

```diff
@@ -0,0 +1,9 @@
+defmodule LivebookWeb.Cookies do
```

Comment:
This rather belongs to the `ZTA.` namespace, otherwise it's quite out of context. We should also have a behaviour like `IdentityProvider` that would implement

@josevalim should `ZTA` be `Identity` then?

Reply:
@jonatanklosko to give you the full context, `ZTA` will be a separate package. So we are keeping only select things in there. :) This is Livebook specific but it implements the ZTA contract.